src/components/atoms/DeezerPlayer.jsx

```diff
@@ -33,59 +39,14 @@ export const DeezerPlayer = ({
   autoplay = false,
   theme = 'DARK',
   color = 'ff0000',
+  isSidePreview,
 }) => {
   let trueUrl, temp, type, id
 
-  const typeAllow = ['artist', 'playlist', 'track', 'album', 'podcast']
-  const typeExist = type => typeAllow.some(el => el === type)
-
-  // from ==>
-  // https://www.deezer.com/track/854470322?utm_source=deezer&utm_content=track-854470322&utm_term=3526920284_1584286559&utm_medium=web
-  // to ==>
-  // https://www.deezer.com/plugins/player?playlist=true&color=${color}&layout=${theme}&type=${type}&id=${url}&app_id=1
-
-  // https://www.deezer.com/plugins/player?format=classic&autoplay=false&playlist=true&color=ff0000&layout=dark&type=track&id=85447032&app_id=1
-
-  /* 
-
-  artist,
-  playlist,
-  track,
-  album,
-  podcast
-
-  */
-
-  // track and playlist
-  // v1
-  // https://www.deezer.com/track/692002952?utm_source=deezer&utm_content=track-692002952&utm_term=3526920284_1584311474&utm_medium=web
-
-  // length: 5
-
-  // v2.2
-  // https://www.deezer.com/playlist/7371286864
+  console.log({ format, isSidePreview })
 
-  // length: 5
-
-  // v2
-  // https://www.deezer.com/ru/playlist/7371286864
-
-  // length: 6
-
-  // vE https://www.deezer.com/plugins/player?format=classic&playlist=true&autoplay=false&color=ff0000&layout=dark&type=playlist&id=7371286864&app_id=1
-
-  // artist
-  // v1 https://www.deezer.com/artist/15888?utm_source=deezer&utm_content=artist-15888&utm_term=3526920284_1584368056&utm_medium=web
-
-  // v2 https://www.deezer.com/ru/artist/15888
-
-  // works
-  // https://www.deezer.com/plugins/player?format=classic&playlist=true&autoplay=false&color=ff0000&layout=dark&type=playlist&id=7371286864&app_id=1
-
-  // e2
-  // https://www.deezer.com/plugins/player?format=classic&playlist=true&autoplay=false&color=ff0000&layout=dark&type=playlist&id=7371286864&app_id=1
-
-  // https://www.deezer.com/plugins/player?format=classic&autoplay=false&playlist=true&width=700&height=350&color=ff0000&layout=dark&size=medium&type=radio&id=artist-27&app_id=1
+  const typeAllow = ['artist', 'playlist', 'track', 'album', 'podcast']
+  const typeExist = (type) => typeAllow.some((el) => el === type)
 
   if (!isValidUrl(url)) {
     return <> </>
```

Comment:
Is this really needed?

Reply:
I created this a long time ago but I think it is needed, otherwise there might be errors if you just type "deezer" in an input field and it would try to convert url to iframe